mcsema/BC/Function.cpp

```diff
@@ -428,6 +428,21 @@ static void LiftBlockIntoFunction(TranslationContext &ctx) {
   }
 }
 
+static void AllocStackVars(
+    llvm::BasicBlock *bb,
+    const NativeFunction *cfg_func){
+  auto func = bb->getParent();
+  llvm::IRBuilder<> ir(bb);
+  const std::list<NativeStackVariable *> &stack_vars = cfg_func->stack_vars;
+  for (auto s : stack_vars){
+    auto char_type = llvm::Type::getInt8Ty(func->getContext());
+    auto int_type = llvm::Type::getInt64Ty(func->getContext());
+    auto value = llvm::ConstantInt::get(int_type, s->get_size());
+    LOG(INFO) << "Inserting var " << s->get_name() << ", size " << s->get_size() << ", Func name " << cfg_func->name << " " << std::endl;
```

Comment:
For the most part, I've been avoiding getters/setters on the `NativeXYZ` types because I just make sure that CFG.cpp correctly initializes all fields, and then they are constant thereafter.